otcdocstheme/theme/otcdocs/static/css/otcdocstheme.css

```diff
@@ -395,7 +395,10 @@ a.headerlink:hover {
   color: var(--dt-color-blue-60);
 }
 
-h1,
+h1 {
+  font-weight: 400;
```

Comment:
let's just have "lighter" here (like in the other case)

Reply:
I liked the behavior in kubernetes doc when the top heading is a bit bolder:
https://kubernetes.io/docs/concepts/overview/what-is-kubernetes/

Check it out. (they overstressed it a bit)